frontend/packages/console-shared/src/components/form-utils/FormFooter.tsx

```diff
@@ -34,11 +38,17 @@ const FormFooter: React.FC<FormFooterProps> = ({
         {infoMessage}
       </Alert>
     )}
-    <ActionGroup className="pf-c-form pf-c-form__group--no-top-margin">
+    <ActionGroup
+      className={cx(
+        'pf-c-form',
+        { 'pf-c-form__group--no-top-margin': noTopMargin },
```

Comment:
It would be better if you just pass a prop from the parent called `actionBarClassNames` and merge them here.